nengo_loihi/builder/connection.py

```diff
@@ -724,23 +724,23 @@ def build_conv2d_connection(model, conn):
     pop_type = 32  # TODO: pick this
     new_transform = copy.copy(conn.transform)
     type(new_transform).init.data[new_transform] = weights
-    weights, indices, axon_to_weight_map, cx_bases = conv2d_loihi_weights(
+    weights, indices, axon_to_weight_map, bases = conv2d_loihi_weights(
```

Comment:
Maybe "offsets" would be better than "bases", since they're essentially offsets for the weight indices.